src/css/angular-tooltips.css

```diff
@@ -36,44 +36,87 @@
   transition: opacity 0.2s linear;
   opacity: 1;
 }
+._720kb-tooltip-caret:before {
+  content:'';
+  position: absolute;
+  width: 0;
+  height: 0;
+  border: 6px solid rgba(0, 0, 0, 0.8);
+}
 ._720kb-tooltip-caret:after {
   content:'';
   position: absolute;
   width: 0;
   height: 0;
+  border: 5px solid rgba(0, 0, 0, 0.8);
 }
-._720kb-tooltip-left ._720kb-tooltip-caret:after {
+._720kb-tooltip-left ._720kb-tooltip-caret:before {
   top: 50%;
   left: 100%;
   margin-left:0;
   margin-top: -6px;
-  border-left: 6px solid rgba(0, 0, 0, 0.8);
-  border-top: 6px solid transparent;
-  border-bottom: 6px solid transparent;
+  border-top-color: transparent !important;
```

Comment:
Sorry but i totally disagree :)

!important is the biggest trouble maker in CSS, should never be used